specs/CustomScrollbar.md

```diff
@@ -0,0 +1,93 @@
+Custom Scrollbar Style
+===
+
+# Background
+Developers would like to be able to set the scrollbar to Windows fluent overlay style and 
+match the Windows theme. Some customers have used browser flags but they change names and behavior and are not reliable. Thus we want
+to provide an extendable API that allows developers to set scrollbar style. Currently, 
+only Windows fluent overlay are supported, adding new styles in the future will not introduce
+any breaking changes.
+
+# Examples
+## WinRT and .NET   
+```c#
+/// Create WebView Environment with option
+async void CreateEnvironmentWithOption()
+{
+    CoreWebView2EnvironmentOptions options = new CoreWebView2EnvironmentOptions();
+    options.CustomScrollbarStyle = ScrollbarStyle.WindowOverlay;
+    CoreWebView2Environment environment = await CoreWebView2Environment.CreateAsync(options: options);
+    webview.EnsureCoreWebView2Async(environment);
+}
+```
+
+## Win32 C++
+```cpp
+void AppWindow::InitializeWebView()
+{
+    auto options = Microsoft::WRL::Make<CoreWebView2EnvironmentOptions>();
+    COREWEBVIEW2_SCROLLBAR_STYLE style = COREWEBVIEW2_SCROLLBAR_STYLE_FLUENT_OVERLAY;
+    CHECK_FAILURE(options->put_CustomScrollbarStyle(style));
```

Comment:
Mismatch with interface definition:
```suggestion
    CHECK_FAILURE(options->put_ScrollbarStyle(style));
```

Reply:
Please update code to match actual names.